source/git-util.js

```diff
@@ -77,6 +84,21 @@ exports.verifyCurrentBranchIsReleaseBranch = async releaseBranch => {
 	}
 };
 
+exports.tagList = async () => {
```

Comment:
This could use a short doc comment on what exactly it returns and the order it returns it.

Reply:
By doc comment, you mean JSDoc or a simple `//` comment?

Reply:
Because a JSDoc comment would be the only one in the file 😆

Reply:
`//`